README.md

```diff
@@ -87,6 +87,10 @@ export class SharedModule { }
 When you lazy load a module, you should use the `forChild` static method to import the `TranslateModule`.
 
 Since lazy loaded modules use a different injector from the rest of your application, you can configure them separately with a different loader/compiler/parser/missing translations handler.
+
+To make a child module extend translations from parent modules use `extend: true`. This will cause the service to also
+use translations from it's parent module.
+
```

Comment:
We should mention that this is working only when the child module is **lazily loaded**.

Reply:
How about something like "To make a lazy loaded child module extend translations from parent modules..."?

Reply:
Yeah, that should work.